python/mlc_chat/rest.py

```diff
@@ -32,7 +32,7 @@ class RestAPIArgs:
             )
         }
     )
-    model_lib_path: str = field(
+    lib_path: str = field(
```

Comment:
@davidpissarra The reason I changed from `lib_path` to `model_lib_path` is because the python API now uses the latter for creating a chat_module object. Shouldn't it be better to remain consistent between python and rest api? We can update the same in the docs for rest_api too